Update README.md (#11)
toledompm/devops-culture: README.md

# DevOps Culture

### WHAT IS

> This project describes the implementation of devops culture in a software development team. The aim of project is to brings students into devops culture by developing an TODO application applying devops techniques such as the develop flow, package, build and deploy flow (Continuous integration and Delivery), software testing documeting the roles involved in the process.

Reviewer: ```suggestion
This project describes the implementation of devops culture in a software development team. The aim of project is to brings students into devops culture by developing a TODO application applying devops techniques such as the development flow, package, build and deploy flow (Continuous integration and Delivery), software testing documenting the roles involved in the process.
```

### TEAM

> * [Jason Davin Oliveira Carneiro](https://www.linkedin.com/in/jason-carneiro/)

Reviewer 2: ```suggestion
* [Jason Carneiro](https://www.linkedin.com/in/jason-carneiro/)
```
:bike:

* [Leonardo Lopes Nunes](https://www.linkedin.com/in/leonardo-lopes/)
* [Marcelo Pereira Fernandes](https://www.linkedin.com/in/marcelo-pereira-fernandes/)
* [Marcos Paulo Mendes Toledo](https://www.linkedin.com/in/marcos-paulo-mendes-toledo-0255b5177/)
* [Nicolas Fernando Espinoza Fujisawa](https://www.linkedin.com/in/nicolas-fernando-56798517b/)

### Repositories
[Application](https://github.com/toledompm/devops-app)

### TECNOLOGIES
- Node.js
- Jest
- Docker
- ECS

> - EC2

Reviewer 2: For now, is not necessary. We can add to application repository

> ### TODO

Reviewer 2: ```suggestion
### Development pillars
```

- [Develop workflow](./config_management/readme.md)
- [Software testing](./tests/readme.md)
- [Environment](./environment/readme.md)

### Sprint 1
- Selection of tools
- Basic definition of versioning processes
- Fundamentals of tests and test types
- Fundamentals of environments and environment types

### Sprint 2
- Basic application building process
- Unit tests and Integration testing
- Cloud Introduction

> - ECS

Reviewer 2: ```suggestion
- Application running in cloud environment
```
